INSTALL

```diff
@@ -5,8 +5,8 @@ qBittorrent - A BitTorrent client in C++ / Qt
 
   - Boost >= 1.65
 
-  - libtorrent-rasterbar >= 1.2.14 (by Arvid Norberg)
-      * https://www.libtorrent.org/
+  - libtorrent-rasterbar ^1.2.14 || >= 2.0.4
```

Comment:
I'm not sure it means exactly what we want...
We support libtorrent 1.2.x (x >= 14) and 2.0.x (x >= 4). Does your record mean exactly that?

Reply:
>We support libtorrent 1.2.x (x >= 14) and 2.0.x (x >= 4). Does your record mean exactly that?
Yes, you can use the link in the commit message to help understand it.

Reply:
>give concrete suggestions

It should mean what I said above:
>We support libtorrent 1.2.x (x >= 14) and 2.0.x (x >= 4).

Reply:
>It should mean what I said above:

If I changed it, are you able to verify I'm correct if you don't understand the notation?

Reply:
>If I changed it, are you able to verify it is correct if you don't understand the notation?

I skimmed through https://semver.npmjs.com/ before commenting on it, although I can't say that I understood it exactly. But one thing is clear to me that the same requirements should look the same. You had different notation for 1.2 and 2.0 branches, which immediately caught my eye.

Reply:
>You had different notation for 1.2 and 2.0 branches, which immediately caught my eye.

Fair, it is amended in the latest push.